pkg/agent/proxy/proxier.go

```diff
@@ -416,6 +427,12 @@ func (p *proxier) installServices() {
 					needUpdateEndpoints = true
 				}
 			}
+			// When the Endpoint is local ClusterIP, it means corresponding local Service has been exported.
```

Comment:
"a local ClusterIP"

Reply:
"the corresponding local Service is a member of the Multi-cluster Service."

Reply:
Done